test/System/Vector2.cpp

```diff
@@ -37,6 +37,48 @@ TEST_CASE("sf::Vector2 class template - [system]")
             CHECK(vector.x == static_cast<int>(sourceVector.x));
             CHECK(vector.y == static_cast<int>(sourceVector.y));
         }
+
+        SUBCASE("Length and angle constructor")
+        {
+            CHECK(sf::Vector2f(0, sf::degrees(0)) == sf::Vector2f(0, 0));
+            CHECK(sf::Vector2f(0, sf::degrees(180)) == sf::Vector2f(0, 0));
```

Comment:
Could we maybe add some arbitrary values for `phi` here (not just half-circles), to see if it's still the zero vector?
Possibly we need `ApproxVec` for rounding errors.

Reply:
Fixed